Assets/Python/RiseAndFall.py

```diff
@@ -3542,119 +3542,120 @@ def create1700ADstartingUnits(self):
 	
 		# China
 		tCapital = tBeijing
-		utils.makeUnit(iMusketman, iChina, tCapital, 12)

```

Comment:
Where did you move the starting units? They don't seem to appear in the scenario file.